engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -515,28 +470,6 @@ public void setRenderBufferMask(FBO fbo, boolean color, boolean normal, boolean
     }
 
     /**
-     * Sets the viewport of the currently bound FBO to the size of the Display.
-     * 
-     * Which might mean the full screen, the full size of the window or the
-     * currently set screenshot size if a screenshot is being taken.
-     */
-    // TODO: verify if this can become part of the bindDisplay() method.
-    public void setViewportToWholeDisplay() {
-        glViewport(0, 0, fullScale.width(), fullScale.height());
-    }
-
-    /**
-     * Sets the viewport of the currently bound FBO to the dimensions of the FBO
-     * given as parameter.
-     *
-     * @param fbo The FBO whose dimensions will be matched by the viewport of the currently bound FBO.
-     */
-    // TODO: verify if this can become part of the FBO.bind() method.
-    public void setViewportToSizeOf(FBO fbo) {
-        glViewport(0, 0, fbo.width(), fbo.height());
-    }
-
-    /**
      * Unbinds any currently bound FBO and binds the default Frame Buffer,
      * which is usually the Display (be it the full screen or a window).
      */
```

Comment:
Please copy this javadoc to bindDisplay() in OpenGLUtils. Can bindDisplay() already be moved completely? Or do you prefer to leave here to avoid touching too many things?